documentation/document.css

```diff
@@ -59,8 +59,6 @@
         width: 25%;
```

Comment:
This may cause overflow when the device screen is small,

Reply:
I forgot to remove it, as afterwards I used flex. Thank you